### PR TITLE
feat(i18n): add support for links in translations

### PR DESCRIPTION
**Pull request by gund**
- [x] Add transformation of links to a default `i18nInjectable`
- [x] Add support for HTML in `i18n` directive
- [x] Implement unit tests

Closes [HRZ-1998](https://spryker.atlassian.net/browse/HRZ-1998)

[HRZ-1998]: https://spryker.atlassian.net/browse/HRZ-1998?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/base/utilities/src/directives/i18n/i18n.directive.ts [hunk 1]
```diff
@@ -33,6 +35,24 @@ import { i18nInjectable, InferI18nContext } from '../../injectables';
...
+    } else {
+      return result.text;
+    }
+  }
+
+  return result;
+}
+
+function asUnsafeHtml(text: string) {
+  return html`${unsafeHTML(text)}`;
```
Comment (supproduction) [inline libs/base/utilities/src/directives/i18n/i18n.directive.ts]: Do we need separate function for it?

File: libs/platform/i18n/project.json [hunk 2]
```diff
@@ -7,7 +7,12 @@
       "executor": "./tools/executors/components-library:build"
     },
     "test": {
-      "executor": "./tools/executors/oryx-vite:vitest"
+      "executor": "./tools/executors/oryx-vite:vitest",
+      "configurations": {
+        "watch": {
+          "watch": true
+        }
+      }
```
Comment (supproduction) [inline libs/platform/i18n/project.json:15]: why do we need it? 
we can just use `--watch`

**Review by supproduction — CHANGES_REQUESTED** [review]

File: libs/base/utilities/src/directives/i18n/i18n.directive.ts (hunk 1 above)
Comment (gund) [reply libs/base/utilities/src/directives/i18n/i18n.directive.ts]: Yeah I previously had 2 places where I was using it but after refactoring it's not needed so will remove it 👌

File: libs/platform/i18n/project.json (hunk 2 above)
Comment (gund) [reply libs/platform/i18n/project.json:15]: Well I'm using NX Console extension to launch nx commands and it does not support writing extra flags so it's nice to have a shortcut to be able to launch watch mode in one click 😊

**Review by supproduction — DISMISSED** [review]

**Review by dunqan — DISMISSED** [review]

**Review by dunqan — DISMISSED** [review]

**Review by dunqan — APPROVED** [review]

**Pull request merged**